service/src/main/java/org/apache/griffin/core/job/LivyTaskSubmitHelper.java

```diff
@@ -0,0 +1,196 @@
+package org.apache.griffin.core.job;
```

Comment:
like the other source code files, license header is required.

Reply:
OK, I added .